Tools/CISupport/ews-build-webserver/master.cfg

```diff
@@ -0,0 +1,125 @@
+
+
+# This is work-around for Twisted having a small size limit for patches. See https://bugs.webkit.org/show_bug.cgi?id=198851#c5
+from twisted.spread import banana
+banana.SIZE_LIMIT = 100 * 1024 * 1024
+
+# This is work-around for https://bugs.webkit.org/show_bug.cgi?id=222361
+from buildbot.process.buildstep import BuildStep
+BuildStep.warn_deprecated_if_oldstyle_subclass = lambda self, name: None
+
+is_test_mode_enabled = utils.load_password('BUILDBOT_PRODUCTION') is None
+custom_suffix = utils.get_custom_suffix()
+
+# We are using multi-master setup with 2 masters.
+# One master handles UI (web-server), and the other handles the back-end.
+# If test mode is enabled, we use single master setup.
+use_multi_master = not is_test_mode_enabled
+
+c = BuildmasterConfig = {}
+
+if use_multi_master:
+    c['multiMaster'] = True
+    c['mq'] = {
+        'type': 'wamp',
+        'router_url': 'ws://localhost:8080/ws',
+        'realm': 'realm1',
+        'wamp_debug_level': 'info'
+    }
+
+c['www'] = dict(port='tcp:8010:interface=127.0.0.1', allowed_origins=["*"])
+c['www']['custom_templates_dir'] = 'templates'
+c['www']['ui_default_config'] = {
+    'Builders.show_workers_name': True,
+    'Builders.buildFetchLimit': 1000,
+    'Workers.showWorkerBuilders': True,
+}
+
+if not is_test_mode_enabled:
+    c['www']['change_hook_dialects'] = dict(
+        github={
+            'class': events.GitHubEventHandlerNoEdits,
+            'secret': utils.load_password('GITHUB_HOOK_SECRET'),
+            'github_property_whitelist': [
+                'github.number',
+                'github.title',
+                'github.head.ref',
+                'github.head.sha',
+                'github.base.ref',
+                'github.head.repo.full_name',
+                'github.head.user.login',
+            ], 'token': utils.load_password('GITHUB_COM_ACCESS_TOKEN'),
+        },
+    )
+
+    GITHUB_CLIENT_ID = utils.load_password('GITHUB_CLIENT_ID')
+    GITHUB_CLIENT_SECRET = utils.load_password('GITHUB_CLIENT_SECRET')
+    if (not GITHUB_CLIENT_ID) or (not GITHUB_CLIENT_SECRET):
+        print('EWS credentials not found. Please ensure GITHUB_CLIENT_ID and GITHUB_CLIENT_SECRET are configured either in env variables or in passwords.json')
+        sys.exit(1)
+
+    # See https://docs.buildbot.net/current/manual/configuration/www.html#example-configs
+    authz = util.Authz(
+        allowRules=[util.AnyEndpointMatcher(role='Buildbot-Administrators', defaultDeny=False),
+                    util.RebuildBuildEndpointMatcher(role='Contributors'),
+                    util.StopBuildEndpointMatcher(role='Committers'),
+                    util.AnyControlEndpointMatcher(role='Buildbot-Administrators')],
+        roleMatchers=[util.RolesFromGroups(groupPrefix='WebKit/')]
+    )
+    auth = util.GitHubAuth(GITHUB_CLIENT_ID, GITHUB_CLIENT_SECRET, apiVersion=4, getTeamsMembership=True, debug=True)
+    #c['www']['auth'] = auth
```

Comment:
Why are we commenting this out?

Reply:
It was from my debugging, will remove. Thanks for noticing.